File: recursie/recursie.pas
program recursie(input,output);    
                                   
var i,j:integer;                   
                                   
procedure loop;                    
 begin                             
   writeln(j,' * ',i,' = ',i*j);
   inc(j);                         
   if j<=10 then loop              
 end;                              
                                   
begin                              
  write('Welke tafel ?:');
  readln(i);                       
  j:=1;                            
  loop                             
end.

File: recursie/recursie.c
#include <stdio.h>
int i,j=1;

void loop(void)
 {
	 printf("\n%d*%d=%d",j,i,i*j);
	 j++;
	 if (j<=10) loop();
 };

void main(void)
{
    printf("\nWelke tafel? :");
	scanf("%d",&i);
	loop();
}
